views/index.handlebars

```diff
@@ -31,5 +31,32 @@
         <p>There are no items yet. Click "List New Item" to add one.</p>
     {{/if}}
 {{else}}
-    <p>This is the about page.</p>
+    <hr>
```

Comment:
Do we need this hr? there is nothing else on the page

Reply:
It's personal design preference to divide it by sections if you really don't like it I could get rid of it.